### PR TITLE
Add Bramble API for toggling whitspace view, part of mozilla/thimble.mozilla.org#1763 - continue

### PR DESCRIPTION
**Pull request by tonypark0403**
Refer : thimble-issue-1763
Implemented white-space-indicator.
I lost my repository by mistake.

### CODE REVIEW

**Comment by tonypark0403** [comment]
@humphd Thank you for helping me. Please check the event parts.

**Comment by tonypark0403** [comment]
@flukeout, @humphd I think it is ready to review.

**Comment by tonypark0403** [comment]
@flukeout, @humphd I am wondering if there is something wrong because it is still open.
Do I have to push the close button?

**Comment by humphd** [comment]
This still needs review.  Don't close it if you want it reviewed.

File: src/command/Commands.js [hunk 1]
```diff
@@ -111,6 +111,7 @@ define(function (require, exports, module) {
     exports.TOGGLE_ACTIVE_LINE          = "view.toggleActiveLine";      // EditorOptionHandlers.js      _getToggler()
     exports.TOGGLE_WORD_WRAP            = "view.toggleWordWrap";        // EditorOptionHandlers.js      _getToggler()
     exports.TOGGLE_ALLOW_JAVASCRIPT     = "cmd.toggleAllowJavaScript";  // EditorOptionsHandlers.js     _getToggler()
+    exports.TOGGLE_ALLOW_WHITESPACE     = "cmd.toggleAllowWhiteSpace";  // EditorOptionsHandlers.js     _getToggler()
```
Comment (humphd) [inline src/command/Commands.js]: This change is unnecessary, and does nothing.  You can remove it with:

```
git checkout master src/command/Commands.js
```

File: src/editor/Editor.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ define(function (require, exports, module) {
         WORD_WRAP           = "wordWrap",
         INDENT_LINE_COMMENT = "indentLineComment",
         ALLOW_JAVASCRIPT    = "allowJavaScript",
+        ALLOW_WHITESPACE    = "allowWhiteSpace",
```
Comment (humphd) [inline src/editor/Editor.js]: Same with this file, none of these changes are necessary.  Please revert them with:

```
git checkout master src/editor/Editor.js
```

**Comment by tonypark0403** [comment]
Hello @humphd, I removed the unnecessary files.

**Pull request merged**